test/pcs/c1/PageMods.test.js

```diff
@@ -0,0 +1,131 @@
+import assert from 'assert'
+import { c1 } from '../../../build/wikimedia-page-library-pcs'
+import domino from 'domino'
+
+const PageMod = c1.PageMods
+const Platforms = c1.Platforms
+const Themes = c1.Themes
+
+describe('pcs.c1.PageMod', () => {
+  const emptyHTML = '<html lang="en"><head><title>Foo</title></head><body><p></p></body></html>'
+
+  describe('.onPageLoad()', () => {
+    it('any', () => {
```

Comment:
I'm not sure I understand this test.  Is it just demonstrating that onPageLoad doesn't throw?

Reply:
Yes, it's mainly to show that it can be called the expected way. Checking the lazy loading or table collapsing event handling seems too complex for this.